### PR TITLE
CRLF to LF and a 5th line

### PR DESCRIPTION
**Pull request by nadimalabbas**
2nd pr body

### CODE REVIEW

**Review by nadtesttest — CHANGES_REQUESTED** [review]
delete 5th line

File: new.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-A new line for PR
-A 2nd line
-a 3rd line
-A 4th line for synchronize
+A new line for PR
+A 2nd line
+a 3rd line
+A 4th line for synchronize
+A 5th line
```
Comment (nadtesttest) [inline new.txt:5]: this line delete it

Comment (nadimalabbas) [reply new.txt:5]: comment by the author

**Comment by nadtesttest** [comment]
comment that does nothing

**Pull request closed without merging**